2.0/README.md

```diff
@@ -0,0 +1,137 @@
+Ruby for general usage and OpenShift - Docker images
```

Comment:
+1 since its in the openshift namespace on GitHub we can skip the OpenShift in the header.
@hhorak could you change it in all the repos ?